src/extractors/gitmojiextractor.py

```diff
@@ -0,0 +1,56 @@
+"""
```

Comment:
I don't think this comment adds anything. What do you say?

Reply:
On second thought, that doesn't really add anything.